specs/lib/getStatisticsSpec.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+var fs = require('fs');
+var addDefaults = require('../../lib/addDefaults');
+var addPipelineExtras = require('../../lib/addPipelineExtras');
+var getStatistics = require('../../lib/getStatistics');
+var readGltf = require('../../lib/readGltf');
+
+describe('getStatistics', function() {
+    it('should return stats for simple box test', function(done){
+        expect(readGltf('specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf')
+            .then(function(gltf) {
+                var stats = getStatistics(gltf);
+
+                expect(stats.buffersSizeInBytes).toEqual(840);
+                expect(stats.numberOfImages).toEqual(1);
+                expect(stats.numberOfExternalRequests).toEqual(4);
+
+                expect(stats.numberOfDrawCalls).toEqual(1);
+                expect(stats.numberOfRenderedPrimitives).toEqual(0);
```

Comment:
This should be a warning sign that something isn't right.

In `getStatistics` `primitive.primitive` should be `primitive.mode`. I believe this is legacy from gltf 0.8.